### PR TITLE
Add Codecov Report Generation

### PR DESCRIPTION
**Pull request by SasinduDilshara**
Add Codecov Report Generation Job to GitHub Actions 
>pull-request.yml and build-master.yml

### CODE REVIEW

**Review by Kishanthan — COMMENTED** [review]
The commit msg exceeds the 50 column limit. Please update the commit message length.

**Comment by SasinduDilshara** [comment]
> The commit msg exceeds the 50 column limit. Please update the commit message length. Commit - [ceb7060](https://github.com/ballerina-platform/module-ballerina-udp/commit/ceb7060029ec775163380ea86a3c942707ab4ec5)

I updated the commit messages

File: build.gradle [hunk 1]
```diff
@@ -100,3 +100,4 @@ release {
 task build {
     dependsOn('udp-ballerina:build')
 }
+
```
Comment (ThisaruGuruge) [inline build.gradle:103]: Extra new line?

**Review by ThisaruGuruge — APPROVED** [review]
LGTM

**Review by Kishanthan — APPROVED** [review]

**Pull request merged**